google-cloud-core/src/com/google/cloud/tools/intellij/analytics/UsageTrackerNotification.java

```diff
@@ -46,29 +46,26 @@ public static UsageTrackerNotification getInstance() {
   /** Show the notification panel. */
   public void showNotification() {
     NotificationListener listener =
-        new NotificationListener() {
-          @Override
-          public void hyperlinkUpdate(Notification notification, HyperlinkEvent event) {
-            if (event.getEventType() == HyperlinkEvent.EventType.ACTIVATED) {
-              final String description = event.getDescription();
-              if ("allow".equals(description)) {
-                usageTrackingManagementService.setTrackingPreference(true);
-                notification.expire();
-              } else if ("decline".equals(description)) {
-                usageTrackingManagementService.setTrackingPreference(false);
-                notification.expire();
-              } else if ("policy".equals(description)) {
-                try {
-                  BrowserUtil.browse(new URL(UsageTrackerPanel.PRIVACY_POLICY_URL));
-                } catch (MalformedURLException ex) {
-                  LOG.error(ex);
-                }
-                notification.expire();
-              } else if ("settings".equals(description)) {
-                final ShowSettingsUtil util = ShowSettingsUtil.getInstance();
-                util.showSettingsDialog(null, UsageTrackerConfigurable.class);
-                notification.expire();
+        (notification, event) -> {
```

Comment:
I think this is the case, but just to make sure I didn't miss anything -> this is just a formatting change (and lambda refactoring) right?

Reply:
right